package.json

```diff
@@ -43,6 +43,7 @@
     "@hapi/yar": "^11.0.2",
     "@joi/date": "^2.1.0",
     "@smithy/node-http-handler": "^2.0.4",
+    "axios": "^1.8.4",
```

Comment:
I didn't need to add this explicitly in the import app. It is a dependency of Notify, and if they change to something else, we'll also need to change, so we shouldn't need to manage our own version.